### PR TITLE
Fix for #277

### PR DESCRIPTION
**Pull request by tmorehouse**

### CODE REVIEW

File: lib/components/form-fieldset.vue [hunk 1]
```diff
@@ -27,12 +38,15 @@
                 return this.horizontal ? ('col-sm-' + (12 - this.labelSize)) : 'col-12';
             }
         },
-        mounted() {
-            const content = this.$refs.content;
-            if (!content) {
-                return;
+        methods: {
+            target() {
```
Comment (pi0) [inline lib/components/form-fieldset.vue]: Can't we implement and use target as a computed property?

Comment (tmorehouse) [inline lib/components/form-fieldset.vue]: Hmmm, would the this.$refs.content exist when the prop is calculated?

Comment (tmorehouse) [inline lib/components/form-fieldset.vue]: We can try as a computed property, and see how it works.

Comment (tmorehouse) [inline lib/components/form-fieldset.vue]: I could also alter the default `inputSelector` value to include `.dropdown` and the check for its ID existance. And in dropdown, we could generate a default `b_UID` if an ID is not given.

Comment (tmorehouse) [inline lib/components/form-fieldset.vue]: Would maybe placing the target computation in `mounted()` and `updated()` be safer (and have target as a data prop)?

Comment (pi0) [inline lib/components/form-fieldset.vue]: The thing I'm sure $refs is only accessible​ after mounted() so we may keep it as a method but just call it inside them :)

Comment (tmorehouse) [inline lib/components/form-fieldset.vue]: OK so leave it as a method, and then have `mounted()` and `updated()`call the method and update a data value, which is then referenced by the the `<label>`'s `:for`

**Comment by tmorehouse** [comment]
Once this is merged, I will add a default ID to the `.dropdown` if one is not provided to the dropdown component.

File: lib/components/form-fieldset.vue [hunk 2]
```diff
@@ -27,12 +39,21 @@
...
-            if (!content) {
-                return;
+        methods: {
+            updateTarget() {
+                const content = this.$refs.content;
+                if (!content) {
+                    return null;
+                }
+                const input = content.querySelector(this.inputSelector);
+                return (input && input.id) ? input.id : null;
```
Comment (pi0) [inline lib/components/form-fieldset.vue]: Shouldn't we update some data here? :))

Comment (tmorehouse) [inline lib/components/form-fieldset.vue]: Ah crap.  My fingers are too fast and not listening to my head :p

**Comment by tmorehouse** [comment]
So if server side rendering is used, can we actually use  the DOM function `this.$refs.content.querySelector(...)`?  I haven't played much with server side rendering.

**Comment by pi0** [comment]
@thomasfowler As long as it is not happening during first render (computed and data bindings) SSR should be safe. I'll check that before release too.

**Pull request merged**